src/entity/posts/Post.ts

```diff
@@ -259,7 +272,7 @@ export class Post {
   slug: string;
 
   @Column({ type: 'jsonb', default: {} })
-  contentMeta: object;
+  contentMeta: PostContentMeta;
```

Comment:
I did not want this typed on purpose since it has a lot more stuff that API does not use so this implies it has only above fields. With `object` it requires manual casting when used which is I think better in this case.

Reply:
👍

Reply:
Reverted to object and use type casting instead